### PR TITLE
Add test for checking text-decoration-skip default on body

### PR DESCRIPTION
**Pull request by drott**
As proposed on the WHATWG HTML living standard as a change
to the default UA stylesheet.

### CODE REVIEW

**Comment by drott** [comment]
The corresponding WHATWG PR is here: https://github.com/whatwg/html/pull/3102

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19241)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/284240254)

**Comment by sideshowbarker** [comment]
@drott Sorry for the trouble, but please rebase your PR branch against the master branch here. That’s necessary to pull in 8f30758 to fix the Travis error at https://travis-ci.org/w3c/web-platform-tests/jobs/284240257 that’s otherwise going to block merging of this.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (Ms2ger) [inline html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Stray end tag.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (Ms2ger) [inline html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: I don't think you need this; adding tests should be fine up to and including the firing of the load event.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (Ms2ger) [inline html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: That being said, I'm not sure if this even needs to wait for the load event.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (Ms2ger) [inline html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Does the spec not define the order? Please file a bug on the spec if so, and link to it here.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (Ms2ger) [inline html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Nit: while you're dealing with my other complaints, none of the stuff in the head particularly needs to be indented.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (Ms2ger) [inline html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Depending on the outcome of the spec discussion, this might belong in a CSS spec folder.

**Review by Ms2ger — DISMISSED** [review]
Thanks for the test!

**Comment by drott** [comment]
I'll update this according to https://github.com/w3c/csswg-drafts/commit/70e40c9315bf8d478ddecf0da5e5af3ddacbc509 (moving to text-decoration-skip-ink: auto= and incorporate the review feedback.

CC @domenic

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (drott) [reply html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Moved to css-text-decor.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (drott) [reply html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Header indentation removed.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (drott) [reply html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Removed.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (drott) [reply html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Calling the test function inline now.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (drott) [reply html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: Not applicable anymore since the new property is not multi-valued.

File: html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html
Comment (drott) [reply html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html]: See above, calling the test function inline now.

**Comment by drott** [comment]
Moved to `css/css-text-decor/text-decoration-skip-ink.html` rebased and most review comments addressed, PTAL.

**Review by domenic — APPROVED** [review]
LGTM

**Comment by drott** [comment]
It seems I can't merge without an LGTM from you, @Ms2ger - would you mind taking another look?

**Comment by drott** [comment]
Thanks for the quick response.

**Pull request merged**
